Twig/Extension/MediaExtension.php

```diff
@@ -169,7 +169,7 @@ public function thumbnail($media = null, $format, $options = array())
 
         $options = array_merge($defaultOptions, $options);
 
-        $options['src'] = $provider->generatePublicUrl($media, $format);
+        $options = $provider->getHelperProperties($media, $format, $options);
```

Comment:
> @greg0ire it was in my beginning and tests were added which were all green

Yeah plus the tests look good and so does the commit message, so I can understand why you merged, I might have done the same had I seen that. We must not hesitate to ask clarifications when we see strange things like that.

@elvetemedve answered about this on #1065 , he told us "Since it's FOSS software, feel free to contribute. ;)". So basically, we're on our own. The problem is, I have no clue how to undo this without creating other BC-breaks, or what the proper fix.